### ISSUE TITLE
Please fix

### ISSUE THREAD

**Issue by ThorvaldAagaard** [EdwardPiwowar/BBA#213]
0304C74937D3C340B1AB40297BCA
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/2e5bbbc7-a7f4-4a5c-9244-ca2ff1a199d4)

Blackwood for clubs is wrong, and then bidding a grand missing an ace

**Comment by EdwardPiwowar** [comment]
Yes, in this situation where E can only have one heart and it can only be 6-1, should hearts or NT be presumed?

**Comment by ThorvaldAagaard** [comment]
After 4H hearts should be presumed

**Comment by EdwardPiwowar** [comment]
Thx

**Comment by ThePokerDude** [comment]
8623 goes crazy in this board. It does not find the 3h forcing bid but bids 6h.
![image](https://github.com/user-attachments/assets/0dfe13b5-056f-4109-b224-cf1bb0c31d15)

**Comment by EdwardPiwowar** [comment]
Next BBA
![image](https://github.com/user-attachments/assets/28fd36cd-e22b-4db5-871a-b664d57bd322)
Bidding is more elegant but the bad distribution of diamonds honors spoils the contract.

**Comment by ThorvaldAagaard** [comment]
Whenever you look at a contract just focus on the two hands, with bad connections it will go down on a spade lead. But this is OK.
